python/openmldb_sdk/tests/dbapi_parameterized_test.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# Copyright 2021 4Paradigm
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import os
+from openmldb.dbapi import connect
+import pytest
+from datetime import datetime
+
+from .case_conf import OpenMLDB_ZK_CLUSTER, OpenMLDB_ZK_PATH
+
+
+class TestOpenmldbDBAPI:
```

Comment:
Test class name should be the same with file name

Reply:
updated